pkgs/tools/security/rainbowcrack/default.nix

```diff
@@ -0,0 +1,33 @@
+let
+  pkgs = import <nixpkgs> {  };
+
+in
+pkgs.stdenv.mkDerivation rec {
+  pname = "rainbowcrack";
+  version = "1.8-0kali1";
+
+  src = pkgs.fetchFromGitLab {
+    group = "kalilinux";
+    owner = "packages";
+    repo = "rainbowcrack";
+    rev = "kali/${version}";
+    hash = "sha256-y6jdFVQSO80YLLOitPAY8TVMUsi58B6+3bFoncwr+Qo=";
+  };
+
+  passthru = {
+      NIXPKGS_ALLOW_UNFREE = "1";
+  };
```

Comment:
I'm not entirely sure what was the original purpose of this, but I'm pretty sure this is strictly illegal in nixpkgs.